org.eclipse.lsp4e/src/org/eclipse/lsp4e/LanguageServiceAccessor.java

```diff
@@ -144,13 +144,20 @@ public boolean isActive() {
 
 	public static @NonNull List<CompletableFuture<LanguageServer>> getInitializedLanguageServers(@NonNull IFile file,
 			@Nullable Predicate<ServerCapabilities> request) throws IOException {
+			return getInitializedLanguageServers(file, request, true);
+	}
```

Comment:
this method is now redundant, so I could deprecate it

Reply:
Yes please `@Deprecate(forRemoval=true)`